examples/node-speak-live/index.js

```diff
@@ -0,0 +1,44 @@
+const { createClient, LiveTTSEvents } = require("../../dist/main/index");
+const fetch = require("cross-fetch");
+
+const live = async () => {
+  const text = "Hello, how can I help you today?";
+
+  const deepgram = createClient(process.env.DEEPGRAM_API_KEY, {
+    global: { fetch: { options: { url: "https://api.beta.deepgram.com" } } },
```

Comment:
```suggestion
    global: { websocket: { options: { url: "wss://api.beta.deepgram.com" } } },
```

It seems to look for a websocket override, not a fetch override.